frontend/src/app/(protected)/experiments/add/page.tsx

```diff
@@ -34,10 +36,11 @@ export default function NewExperiment() {
   type Methods = typeof AllSteps;
 
   const [steps, setSteps] = useState(AllSteps[experimentState.methodType]);
+  const [isSubmitting, setIsSubmitting] = useState(false);
```

Comment:
Sometimes when post takes a while, button is still active and you may end up clicking it twice.